cfme/tests/infrastructure/test_vm_engine_relationship.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    pytest.mark.tier(2),
+    pytest.mark.provider(classes=[InfraProvider], selector=ONE_PER_TYPE),
+    pytest.mark.usefixtures('setup_provider'),
+    test_requirements.rhev,
+]
+
+
+@pytest.fixture()
+def new_vm(provider):
```

Comment:
**Question** Reason we can't use the existing `_create_vm` fixture:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/vm.py#L12

Reply:
hmm, it looks like a private method for the `full_template_vm` and `full_template_vm_modscope`. Not sure that's what I need.

Reply:
I thought about it and I don't think I need to provision a VM for this test at all... all I need is to test the marking relationship in the UI, so I can use any existing VM